tests/spec/util/pagination-utils.spec.ts

```diff
@@ -5,41 +5,17 @@ import { loadMoreDataWhenScrolled } from '../../../react-jhipster';
 describe('loadMoreDataWhenScrolled', () => {
   const setLinks = (first, last, prev) => ({ first, last, prev });
   const state = { entities: [{ 1: 'fake1' }, { 2: 'fake2' }, { 3: 'fake3' }] };
-  const payload = { data: [{ 4: 'fake4' }, { 5: 'fake5' }] };
-  const otherPayload = { data: [{ 7: 'fake7' }, { 8: 'fake8' }, { 9: 'fake9' }] };
-  const itemsPerPage = 2;
+  const payload = { data: [{ 1: 'fake4' }, { 2: 'fake2' }, { 3: 'fake3' }] };
 
-  describe('When there is only one page', () => {
-    it('should return payload.data when there is only one page', () => {
-      expect(loadMoreDataWhenScrolled(state.entities, payload.data, setLinks(0, 0, 0), itemsPerPage)).to.eql(payload.data);
+  describe('When sorting/deleting/editing or if there is only one page', () => {
```

Comment:
If I have scrolled to 2nd page and delete/edit something, I would expect the reloaded data to have the first and second page. Can you add a test for that

Reply:
I just tried with Angular and the current data is reset